srcStatic/ModuleLoader.cpp

```diff
@@ -40,6 +27,21 @@ void ModuleLoader::RegisterExternalModules()
 	}
 }
 
+bool ModuleLoader::IsInternalModuleRequested(const std::string& moduleName)
+{
+	const auto isModuleRequested = ToLower(GetOutpost2IniSetting("InternalModules", moduleName));
+
+	if (isModuleRequested == "yes") {
+		return true;
+	}
+	else if (isModuleRequested == "no") {
+		return false;
+	}
+
+	PostErrorMessage(__FILE__, __LINE__, "Internal Module named " + moduleName + " is either missing from the .ini file or contains an innapropriate setting. It must be set to Yes or No");
```

Comment:
We should probably only log an error for a bad value, not for a non-existent value. If we log for non-existent values, then commenting the line to disable a module would generate an error message.

Reply:
I was thinking about this after committing and came to the same conclusion. I'll change in the near future.